src/podcast/feed_sync.py

```
@@ -0,0 +1,318 @@
+                last_checked=datetime.utcnow(),
+            )
+
+            if new_count > 0:
+                self.repository.update_podcast(
+                    podcast_id,
+                    last_new_episode=datetime.utcnow(),
+                )
+
+            logger.info(f"Sync complete for '{podcast.title}': {new_count} new episodes")
+
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)